checkout_cache_criteria.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "type": "application/json",
+    "max_size": 12314
```

Comment:
Since there's a small number of criteria, consider making these environment vairables, e.g. DSS_CHECKOUT_CACHE_FILE_TYPE, DSS_CHECKOUT_CACHE_FILE_SIZE.

Reply:
negative ghost-rider, there can be different content-types and max-sizes based on the types dept on what goes into the datastore.